adapter/config/ingress/controller.go

```diff
@@ -62,10 +62,10 @@ func NewController(client kubernetes.Interface, mesh *proxyconfig.ProxyMeshConfi
 	informer := cache.NewSharedIndexInformer(
 		&cache.ListWatch{
 			ListFunc: func(opts meta_v1.ListOptions) (runtime.Object, error) {
-				return client.ExtensionsV1beta1().Ingresses(options.Namespace).List(opts)
+				return client.ExtensionsV1beta1().Ingresses(meta_v1.NamespaceAll).List(opts)
```

Comment:
can you please make this customizable? It's a matter of passing through the namespace to controllers through the code.